docs/pages/design_tokens.js

```diff
@@ -19,18 +20,32 @@ export type Token = {|
 const tokenCategories = [
   { name: 'Spacing', category: 'spacing', id: 'space' },
   { name: 'Background color', category: 'background-color', id: 'background' },
-  { name: 'Text color', category: 'text-color', id: 'text' },
+  { name: 'Text color', category: 'text-color', id: 'color-text' },
   { name: 'Font size', category: 'font-size', id: 'font-size' },
   { name: 'Font weight', category: 'font-weight', id: 'font-weight' },
   { name: 'Font family', category: 'font-family', id: 'font-family' },
   { name: 'Border color', category: 'color-border', id: 'color-border' },
   { name: 'Elevation', category: 'elevation', id: 'elevation' },
+  { name: 'Data visualization', category: 'data-visualization', id: 'data-visualization' },
 ];
 
-const darkValueCategories = ['Background color', 'Elevation', 'Border color', 'Text color'];
+const darkValueCategories = [
+  'Background color',
+  'Elevation',
+  'Border color',
+  'Text color',
+  'Data visualization',
+];
 
 const headers = ['CSS token name', 'Value', 'Dark mode value', 'Example'];
 
+const sortedDataTokens = dataVizTokens.sort((a, b) =>
```

Comment:
Due to funky sorting, this is needed to show numbers 01-12 in order

Reply:
Consider leaving this as a comment in the code for Future Us